### PR TITLE
Return teardown function from observe

### PR DESCRIPTION
**Pull request by NekR**
I'm making this PR but I'm not 100% sure this is the best way handle this.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,5 +5,8 @@ export default (element, handler) => {
 	frame.style.cssText = CSS;
 	element.appendChild(frame);
 	frame.contentWindow.onresize = () => { handler(); };
-	return frame;
+	return () => {
+    iframe.contentWindow.onresize = null;
```
Comment (NekR) [inline index.js]: Oh, did I use spaces instead of tabs? I'll fix it

File: index.js [hunk 2]
```diff
@@ -5,5 +5,8 @@ export default (element, handler) => {
 	frame.style.cssText = CSS;
 	element.appendChild(frame);
 	frame.contentWindow.onresize = () => { handler(); };
-	return frame;
+	return () => {
+		iframe.contentWindow.onresize = null;
```
Comment (NekR) [inline index.js:9]: Oh, sometimes `iframe.contentWindow` could be already `null` at this point.

Comment (developit) [reply index.js:9]: hmm - I wonder if that means the unsetting could be unnecessary?
It'd be `null` if any ancestor was removed.

Comment (NekR) [reply index.js:9]: Yeah, I have such thoughts too. Closing this and just adding notes: https://github.com/developit/simple-element-resize-detector/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R40 ?

Comment (developit) [reply index.js:9]: cool. would be neat to get a definitive answer on this from a browser vendor, though it likely varies. thinking older IE would be the offender.

Comment (fregante) [reply index.js:9]: I think the teardown function should detach the iframe instead, otherwise that will be left behind and nothing will make use of it.

Comment (developit) [reply index.js:9]: Currently we return the frame, which gives you the DOM .remove() method for doing that.

**Pull request closed without merging**